applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -120,18 +122,23 @@ class TrilinosBlockBuilderAndSolver
     ///@name Life Cycle
     ///@{
 
+    /**
+     * @brief Default constructor (empty)
+     */
+    explicit TrilinosBlockBuilderAndSolver() = default;
```

Comment:
Just for my curiosity, are you adding it because the one automatically added by the compiler is non-`explicit`?

Reply:
Yes